motion.c

```diff
@@ -894,13 +894,13 @@ static int motion_init(struct context *cnt)
     /* create a reference frame */
     alg_update_reference_frame(cnt, RESET_REF_FRAME);
 
-#if defined(HAVE_LINUX_VIDEODEV_H) && !defined(WITHOUT_V4L) && !defined(__FreeBSD__)
+#if !defined(WITHOUT_V4L2) && !defined(__FreeBSD__)
     /* open video loopback devices if enabled */
     if (cnt->conf.vidpipe) {
         MOTION_LOG(NTC, TYPE_ALL, NO_ERRNO, "%s: Opening video loopback device for normal pictures");
 
         /* vid_startpipe should get the output dimensions */
-        cnt->pipe = vid_startpipe(cnt->conf.vidpipe, cnt->imgs.width, cnt->imgs.height, cnt->imgs.type);
+        cnt->pipe = vid_startpipe(cnt->conf.vidpipe, cnt->imgs.width, cnt->imgs.height, V4L2_PIX_FMT_YUV420);
```

Comment:
Yes that is intentional.  In the old V4L days, it looks like Motion processed images in the format in which they were provided by the device.  The new V4L2 and Netcam way is that we always turn all our images into YUV420 then do the detection, overlays, processing, etc.  This is specifying to our loopback device that we will be sending images as YUV420 regardless of whether source image was RGB, Y11, YUYV, etc.  Ideally, this really just needs to get removed as a parameter.